src/index.html

```diff
@@ -1,13 +1,525 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>NAMU</title>
+    <link
+      rel="shortcut icon"
+      type="image"
+      href="./images/favicon.png"
+    >
+    <link rel="stylesheet" href="https://meyerweb.com/eric/tools/css/reset/reset.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&family=Raleway:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#" class="logo">
+              <img
+                src="./images/NAMUlogo-white.png"
+                alt="logo NAMU"
+                class="logo__image"
+              >
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+
+            <div class="top-actions__language language">
+              <select name="language-select" class="language__select">
+                <option
+                  lang="ua"
+                  value="ukrainian"
+                  selected
+                  class="language__option"
+                >
+                  UA
+                </option>
+                <option
+                  lang="en"
+                  value="english"
+                  class="language__option"
+                >
+                  EN
+                </option>
+              </select>
+            </div>
+          </div>
+
+          <div class="grid grid--tablet grid--desktop">
+            <h1 class="header__title grid__item--tablet-1--5 grid__item--desktop-1--6">
+                Мистецтво ХІХ - ХХ&nbsp;ст.
+            </h1>
+
+            <a href="#visit" class="button header__button grid__item--desktop-1--3 grid__item--tablet-1--3">
+              Купити квиток
+            </a>
+          </div>
+
+          <h6 class="header__dates">10.08 - 10.10</h6>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__top top-actions">
+          <a href="/" class="logo">
```

Comment:
Make sure to change this, cause now If I go ahead to this link, there is a 404 error
```suggestion
          <a href="#" class="logo">
```